volatility3/framework/constants/linux/__init__.py

```diff
@@ -291,5 +291,37 @@
 IFA_LINK = IPV6_ADDR_LINKLOCAL
 IFA_SITE = IPV6_ADDR_SITELOCAL
 
-# Promiscous mode
-IFF_PROMISC = 0x100
+# Only for kernels < 3.15 when the net_device_flags enum didn't exist
+# ref include/uapi/linux/if.h
+NET_DEVICE_FLAGS = {
```

Comment:
This also looks like it could be a python Enum?

Reply:
hm here changing this to an Enum makes the code a bit complicated... have a look at how it's used in `_get_flag_choices()` and `_get_net_device_flag_value()` to support both kernels <3.15 and >=3.15.